.travis.yml

```diff
@@ -28,7 +27,7 @@ script:
     if [[ "$TRAVIS_RUST_VERSION" == nightly* ]]; then
       (cd examples && ./test_nightly)
     else
-      (cd examples && ./test_stable)
+      (cd examples && TRAVIS_RUST_VERSION=$TRAVIS_RUST_VERSION./test_stable)
```

Comment:
Is there a space missing between the variable and the script name? (I'd also explicitly use `env TRAVIS_RUST_…`)